samples/subsys/sensing/simple/src/main.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2023 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys_clock.h>
+#include <zephyr/sensing/sensing.h>
+
+#define MAX_SENSOR_COUNT 128
+LOG_MODULE_REGISTER(main, LOG_LEVEL_INF);
+
+static void acc_data_event_callback(int handle, const void *buf, int size)
+{
+
+}
+
+static void hinge_angle_data_event_callback(int handle, const void *buf, int size)
+{
+
+}
+
+void main(void)
+{
+	const struct sensing_callback_list base_acc_cb_list = {
+		.on_data_event = &acc_data_event_callback,
+	};
+	const struct sensing_callback_list lid_acc_cb_list = {
+		.on_data_event = &acc_data_event_callback,
+	};
+	const struct sensing_callback_list hinge_angle_cb_list = {
+		.on_data_event = &hinge_angle_data_event_callback,
+	};
+	const struct sensing_sensor_info *info[MAX_SENSOR_COUNT];
+	const struct sensing_sensor_info *tmp_info;
+	struct sensing_sensor_config base_acc_config;
+	struct sensing_sensor_config lid_acc_config;
+	struct sensing_sensor_config hingle_angle_config;
+	int base_acc;
+	int lid_acc;
+	int hinge_angle;
+	int ret, i, num = 0;
+
+	ret = sensing_get_sensors(&num, info);
+	if (ret) {
+		LOG_ERR("sensing_get_sensors error");
+		return;
+	}
+
+	for (i = 0; i < num; ++i) {
+		LOG_INF("Sensor %d: name: %s friendly_name: %s type: %d index: %d",
+			 i,
+			 info[i]->name,
+			 info[i]->friendly_name,
+			 info[i]->type,
+			 info[i]->sensor_index);
+	}
+
+	LOG_INF("sensing run successfully");
+
+	ret = sensing_open_sensor_by_type(SENSING_SENSOR_TYPE_MOTION_ACCELEROMETER_3D,
+				0,
+				&base_acc_cb_list,
+				&base_acc);
+	if (ret) {
+		LOG_ERR("sensing_open_sensor_by_type, type:0x%x index:0 error:%d",
+			SENSING_SENSOR_TYPE_MOTION_ACCELEROMETER_3D, ret);
+	} else {
+		base_acc_config.attri = SENSING_SENSOR_ATTRIBUTE_INTERVAL;
+		base_acc_config.interval = 100 * USEC_PER_MSEC;
+		ret = sensing_set_config(base_acc, &base_acc_config, 1);
+		if (ret) {
+			LOG_ERR("base_acc sensing_set_interval error:%d\n", ret);
+		}
+	}
+
+	ret = sensing_open_sensor_by_type(SENSING_SENSOR_TYPE_MOTION_ACCELEROMETER_3D,
+				1,
+				&lid_acc_cb_list,
+				&lid_acc);
+	if (ret) {
+		LOG_ERR("sensing_open_sensor_by_type, type:0x%x index:1 error:%d",
+			SENSING_SENSOR_TYPE_MOTION_ACCELEROMETER_3D, ret);
+	} else {
+		lid_acc_config.attri = SENSING_SENSOR_ATTRIBUTE_INTERVAL;
+		lid_acc_config.interval = 100 * USEC_PER_MSEC;
+		ret = sensing_set_config(base_acc, &lid_acc_config, 1);
```

Comment:
Should we be using `lid_acc` here?

Reply:
You're right, thanks a lot!  will fix it.

Reply:
fixed it, thanks.